restapi/src/test/java/io/stargate/web/docsapi/service/query/FilterExpressionTest.java

```diff
@@ -342,6 +342,32 @@ public void conditionFalse() {
       verifyNoMoreInteractions(condition);
     }
 
+    @Test
+    public void conditionFalsePathSegment() {
```

Comment:
Do we have a unit test for a positive segment match? Sorry, I could not find it.

Reply:
Yea there is no need for this, we wanted to makes sure here that the path match, and I can not do that positive condition.

Reply:
ok :+1: